core/src/main/scala/cats/syntax/parallel.scala

```diff
@@ -32,8 +32,12 @@ trait ParallelSyntax extends TupleParallelSyntax {
   implicit final def catsSyntaxParallelSequence1[T[_]: Traverse, M[_], A](tma: T[M[A]]): ParallelSequenceOps1[T, M, A] =
     new ParallelSequenceOps1(tma)
 
+  @deprecated("Kept for binary compatibility", "2.8.0")
   implicit final def catsSyntaxParallelAp[M[_]: FlatMap, A](ma: M[A]): ParallelApOps[M, A] =
     new ParallelApOps(ma)
+
+  implicit final def catsSyntaxParallelAp1[M[_], A](ma: M[A]): ParallelApOps[M, A] =
+    new ParallelApOps(ma)
```

Comment:
`FlatMap` also didn't get used so have removed it as a constraint for using the `NonEmptyParallel` syntax

Reply:
oh wait, maybe it is..